src/librustdoc/html/render/print_item.rs

```diff
@@ -1458,11 +1460,28 @@ fn document_type_layout(w: &mut Buffer, cx: &Context<'_>, ty_def_id: DefId) {
                     pl = if bytes == 1 { "" } else { "s" },
                 );
             }
-            writeln!(w, "</div>");
         }
-        // Layout errors can occur with valid code, e.g. if you try to get the layout
-        // of a generic type such as `Vec<T>`. In case of a layout error, we just
-        // don't show any layout information.
-        Err(_) => {}
+        // This kind of layout error can occur with valid code, e.g. if you try to
+        // get the layout of a generic type such as `Vec<T>`.
+        Err(LayoutError::Unknown(_)) => {
+            writeln!(
+                w,
+                "<p><strong>Note:</strong> Unable to compute type layout, \
+                 possibly due to this type having generic parameters. \
+                 Layout can only be computed for concrete, fully-instantiated types.</p>"
+            );
+        }
+        // This kind of error probably can't happen with valid code, but we don't
+        // want to panic and prevent the docs from building, so we just let the
+        // user know that we couldn't compute the layout.
+        Err(LayoutError::SizeOverflow(_)) => {
+            writeln!(
+                w,
+                "<p><strong>Note:</strong> Encountered an error during type layout; \
```

Comment:
Just something I was wondering: is there a reason why you used `<strong>` and not `<b>`?

Reply:
Isn't `<strong>` recommended over `<b>`?

Reply:
> Just something I was wondering: is there a reason why you used `<strong>` and not `<b>`?

I thought that in general you're supposed to use `<strong>`, but it looks there are certain cases where `<b>` is supposed to be used.

However, based on MDN's docs, I think this is a [correct usage of `<strong>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#labeling_warnings).